.github/workflows/pre_commit_autoupdate.yml

```diff
@@ -0,0 +1,86 @@
+name: Autoupdate pre-commit config
+
+on:
+  schedule:
+    - cron: "0 0 * * 6"
+
+jobs:
+  pre_commit_autoupdate:
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      config_path: .pre-commit-config.yaml
+      update_message_path: /tmp/message.txt
+      update_branch: chore/pre-commit-autoupdate
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Create or checkout update branch
+        id: create_branch
+        run: |
+          export branch_exists_at_remote=$(git ls-remote --heads origin ${{ env.update_branch }})
```

Comment:
Could you adjust the spaces inside the round brackets to be consistent within this file please?

Some of the later lines include a space after the opening bracket and before the closing bracket. There's one line that mixes these. I don't mind which style you opt for, although the coverage action does include spaces.